include/xeus/xinterpreter.hpp

```diff
@@ -66,6 +66,13 @@ namespace xeus
         void display_data(nl::json data, nl::json metadata, nl::json transient);
         void update_display_data(nl::json data, nl::json metadata, nl::json transient);
         void publish_execution_input(const std::string& code, int execution_count);
+        nl::json publish_execution_output(int execution_count,
```

Comment:
Should this be named `make_reply`? Your implementation does not publish anything, and if I understand correctly your PR, it shouldn't.

Reply:
Also this is not only for the execution request, right? It's for all kinds of requests?